fdbclient/KnobsImpl.cpp

```diff
@@ -23,16 +23,14 @@
 #include "fdbclient/SystemData.h"
 #include "flow/UnitTest.h"
 
-std::unique_ptr<ClientKnobs> globalClientKnobs = std::make_unique<ClientKnobs>();
-ClientKnobs const* CLIENT_KNOBS = globalClientKnobs.get();
-
 #define init(knob, value) initKnob(knob, value, #knob)
```

Comment:
The copyright at the top of this file needs to be changed to reflect the new name of this file.